00-projects-template/Makefile

```
@@ -19,7 +20,7 @@ OBJECTS =	$(SOURCES:.c=.o)
 
 # Main rules
 all: $(OBJECTS)
-	@$(AR) $(LIBRARY) $(OBJECTS)
+	@$(AR) $(LIBRARY) $(OBJECTS) $(INCLUDE)
```

You're already using *Libft* in the `%.o: %.c` rule to create the objects, so the variable isn't needed.

```suggestion
	@$(AR) $(LIBRARY) $(OBJECTS)
```